composer.json

```diff
@@ -29,6 +29,9 @@
             "role": "Previous Maintainer"
         }
     ],
+    "replace": {
+        "mdanter/ecc": "*"
+    },
```

Comment:
Is the `*` a good idea? It basically means that this package acts as a drop-in replacement for _any version_ of `mdanter/ecc` which is certainly not the case.

This should be either `conflict`, meaning that this package and `mdanter/ecc` must not be installed at the same time, or we should pin the last release of the `mdanter/ecc` package here.

Reply:
What precisely is the scenario you are worried about?